x/airdrop/keeper/claim_record.go

```diff
@@ -128,17 +128,14 @@ func (k Keeper) GetClaimableAmountForAction(ctx sdk.Context, chainID string, add
 	if err != nil {
 		return 0, err
 	}
-	if cr.Address == "" {
```

Comment:
Do we validate the address prior to this?

Reply:
yes, this is after `GetClaimRecord` that calls `AccAddressFromBech32`...
thus, an empty address is not possible at this point (previously needed as our initial implementation was based on Stargaze claim module which allowed an empty address at this point).